opus/import/do_dictionary.py

```diff
@@ -31,6 +31,10 @@ def create_import_definitions_table():
     json_def_path = (opus_secrets.DICTIONARY_JSON_SCHEMA_PATH +
                      '/internal_def*.json')
     json_list += glob.glob(json_def_path)
+    # Tooltips for mult 
+    json_tooltips_def_path = (opus_secrets.DICTIONARY_JSON_SCHEMA_PATH +
```

Comment:
I think we want to do the same thing here that's done for obs*.json above . Allow multiple mult_tooltips files like mult_tooltips_mission and mult_tooltips_instrument. Otherwise the file is going to get enormous, and this allows multiple people to work on different widgets at the same time. Since you're already using glob this just means changing the filename to mult_toolips*.json.